### ISSUE TITLE
Instructions for mac installation

### ISSUE THREAD

**Issue by centralswing** [Swirt/typertools-src#10]
to any Mac users looking to use the Typertools linked in #free-resources, download the .dmg file, open it, and you'll have two files.  "Install.cmd" and a folder named "typertools" folder.  Before running the "install.cmd," copy the "typertools," open finder, hold option and go to your Library.   From here go to Adobe -> CEP -> extensions and copy the folder in there.
Then run the "install.cmd," I still got error messages, but when I went into PS, the typertools menu was there under Window->Extensions
It's important to note which "Library" you go in to, you'll know your in the right one if there is nothing else in it before you copy the "typertools" folder (since Mac has like 3 different libraries).  The one you want is the one you get to by holding option -> go -> library in your finder.
~/Library/Application Support/Adobe/CEP/extensions
tadah, the real actual directory to the folder you want lmao, I really suck with directories!!!

From Urbs on SS Discord

**Comment by thiennhing** [comment]
Hello~ how can you run the install.cmd? :( Mac can't run ".cmd" @.@

**Comment by Swirt** [comment]
@thiennhing In version 1.4.3 a mac installer was added. There is an install.sh, launch it from the terminal.

**Comment by thiennhing** [comment]
@Swirt I'm sorry, I don't understand what the terminal say T.T not just open and it will be done, right? Need to do some steps. I try to search "how to run .sh on mac" but I still can't know the way exactly, stuck for hours :(

**Comment by Swirt** [comment]
I don't have a mac, I can only translate other people's words. To install, it should be enough to do the following in the terminal:
`cd <directory of the unpacked archive> ; sh ./install.sh`

**Comment by thiennhing** [comment]
@Swirt thank you for your answer but I can't  hmmm. I gave up...

**Comment by emyarod** [comment]
@Swirt the `install_mac.sh` script included in the release zip needs to have LF line endings. it looks like it currently has CLRF line endings which may cause errors when running `sh ./install_mac.sh`

**Comment by mehzad** [comment]
yeah, that CRLF line endings is the issue with the current mac installation.
i changed that [DOS Format] to [UNIX format] with nano and the script worked for me.

also, about the first post of this thread, i'm really sorry for the mess i made there.
in my defense, i was just helping someone through direct messaging, then that dmg got out and shared in that server.

**Comment by Tiger-Snowy** [comment]
Hi! Was someone finally able to make it work on Mac (M1 specifically)? I looked for hours into it, without success. I finally put the folder directly in CEP/extensions and Typer Tools does appear in Legacy Extensions now, but the window is blank and I have a message error saying that <extension ''Typer Tools'' could not be loaded.>

Of what I understood from the conversation, it would be a problem with the script? Would it be possible to update it then? I don't know a thing about codes ;-;

Thank you in advance for your answer~ 

And thank you a lot for sharing that extension @Swirt , I got to use it a bit when I had my PC and it is really useful 😁

**Comment by pierobisson78** [comment]
> 

mehzad, could you send me your modified script? I keep having issues with the install: there is no app folder in the typertools extension folder and an error message keeps popping when Photoshop is launching. Thanks.

**Comment by pierobisson78** [comment]
> @Swirt the `install_mac.sh` script included in the release zip needs to have LF line endings. it looks like it currently has CLRF line endings which may cause errors when running `sh ./install_mac.sh`

mehzad, could you send me your modified script? I keep having issues with the install: there is no app folder in the typertools extension folder and an error message keeps popping when Photoshop is launching. Thanks.

**Comment by Shifu1977** [comment]
Please any guide how to install typertools on mac os terminal????

Please any help...
